bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -82,10 +82,27 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList',
     update: function() {
       this.element.find('select').trigger('chosen:updated');
     },
-    /**  @method addOption appends a new option to the select
-     *   @param { Object } object with keys id and title
+    /**  
+     * @method hideOptions hide all options except placeholder
+     */
+    hideOptions: function () {
+      this.resetToPlaceholder();
+      var chosen = this.element.find('select');
+      chosen.find('option').not(":first-child").css('display', 'none');
```

Comment:
I'd say this is bound to cause problems. How about we just hide the whole dropdown and possibly replace it with a text message where used if applicable.